vllm/v1/request.py

```diff
@@ -21,9 +33,9 @@ def __init__(
         request_id: str,
         prompt: Optional[str],
         prompt_token_ids: List[int],
-        multi_modal_inputs: Optional[List["MultiModalKwargs"]],
+        multi_modal_inputs: Optional[List[MultiModalKwargs]],
         multi_modal_hashes: Optional[List[str]],
-        multi_modal_placeholders: Optional[List["PlaceholderRange"]],
+        multi_modal_placeholders: Optional[List[PlaceholderRange]],
```

Comment:
What is this change for?

Reply:
It looks like I made this change when cleaning up some `pre-commit` issues. It definitely doesn't seem related to this branch, so it could be reverted if you'd like, though it does seem like a minor improvement. Just let me know if you want it pulled.

Reply:
Oh i think this was from old commit where we touch requests, will revert this.